src/plugins/formvalid/formvalid-en.hbs

```diff
@@ -357,6 +357,22 @@
 				</details>
 			</fieldset>
 
+			<div class="form-group">
+				<label for="cname1" class="required required-no-asterisk"><span class="field-name">Country name</span> (no asterisk) <strong class="required">(required)</strong></label>
```

Comment:
Btw I could add ``fieldset``/``legend`` and standalone ``checkbox`` examples if needed.

Didn't do it from the outset since I felt like they'd amount to bloat. Especially since the "required-no-asterisk" class doesn't have its own properties (it's nothing more than a ``:not()`` pseudo-class).